packages/inifiles/inifiles.2.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+synopsis: "An ini file parser"
+maintainer: ["Emile Trotignon"]
+x-maintenance-intent: ["(latest)"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/ahrefs/inifiles"
```

Comment:
It would probably also be good to correct the homepage link upstream as I can see the latest other changes have reverted it :slightly_smiling_face: